src/neo/Ledger/Blockchain.cs

```diff
@@ -62,7 +62,7 @@ public class RelayResult { public IInventory Inventory; public VerifyResult Resu
         private readonly NeoSystem system;
         private readonly List<UInt256> header_index = new List<UInt256>();
         private uint stored_header_count = 0;
-        private readonly Dictionary<UInt256, Block> block_cache = new Dictionary<UInt256, Block>();
+        private readonly Dictionary<uint, Block> block_cache = new Dictionary<uint, Block>();
```

Comment:
If it is `Dictionary<UInt256, Block>`, then we can use the cache for both overloads of `GetBlock()`.

Reply:
Done.